diagnostic.md

```diff
@@ -32,10 +37,11 @@ Record your responses inside the fenced code blocks below each question.
     });
 
     this.route('product', { path: '/products/:product_id' }); // <= 👀 here
+
     ```
 
     ```md
-    <!-- your response here -->
+      The first one is a nested route
```

Comment:
After class today, do you have a better understanding of this?

These two route definitions are identical  URL paths.
You are correct that one is nested.  Nested route definitions, require a nested file structure.
One of the biggest distinctions is where Ember will render the content.  The nested `'products'/'product'` instructs Ember to render `app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  The non-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.